# StringologyTimes


## Contents
- [arXiv List for Stringologist](https://stringologytimes.github.io/StringologyTimes/docs/output/arxiv_list)
- [Papers for Stringologist (2010)](https://stringologytimes.github.io/StringologyTimes/docs/output/list_2010)
- [Papers for Stringologist (2011)](https://stringologytimes.github.io/StringologyTimes/docs/output/list_2011)
- [Papers for Stringologist (2012)](https://stringologytimes.github.io/StringologyTimes/docs/output/list_2012)
- [Papers for Stringologist (2013)](https://stringologytimes.github.io/StringologyTimes/docs/output/list_2013)
- [Papers for Stringologist (2014)](https://stringologytimes.github.io/StringologyTimes/docs/output/list_2014)
- [Papers for Stringologist (2015)](https://stringologytimes.github.io/StringologyTimes/docs/output/list_2015)
- [Papers for Stringologist (2016)](https://stringologytimes.github.io/StringologyTimes/docs/output/list_2016)
- [Papers for Stringologist (2017)](https://stringologytimes.github.io/StringologyTimes/docs/output/list_2017)
- [Papers for Stringologist (2018)](https://stringologytimes.github.io/StringologyTimes/docs/output/list_2018)
- [Papers for Stringologist (2019)](https://stringologytimes.github.io/StringologyTimes/docs/output/list_2019)
- [Papers for Stringologist (2020)](https://stringologytimes.github.io/StringologyTimes/docs/output/list_2020)
- [Papers for Stringologist (2021)](https://stringologytimes.github.io/StringologyTimes/docs/output/list_2021)
- [Papers for Stringologist (2022)](https://stringologytimes.github.io/StringologyTimes/docs/output/list_2022)
- [All papers for Stringologist](https://stringologytimes.github.io/StringologyTimes/docs/output/complete_list)

## Statistics

![paper_statistics_for_each_year](https://stringologytimes.github.io/StringologyTimes/docs/output/paper_statistics_for_each_year.png)

![checker_table_2029_2020](https://stringologytimes.github.io/StringologyTimes/docs/output/checker_table_2029_2020.png)  
![checker_table_2019_2010](https://stringologytimes.github.io/StringologyTimes/docs/output/checker_table_2019_2010.png)  


## Recent accepted papers for stringologist

### 2022/09/07 [ISAAC 2022](https://isa.hanyang.ac.kr/isaac2022/apl.html)

1. On the Complexity of Tree Edit Distance with Variables  
Tatsuya Akutsu, Tomoya Mori, Naotoshi Nakamura, Satoshi Kozawa, Yuhei Ueno and Thomas N. Sato  
2. Succinct List Indexing in Optimal Time  
William Holland  
3. Simple Order-isomorphic Matching Index with Expected Compact Space  
Sung-Hwan Kim and Hwan-Gue Cho  
4. Computing palindromes on a trie in linear time  
Takuya Mieno, Mitsuru Funakoshi and Shunsuke Inenaga  
5. External-memory dictionaries with worst-case update cost  
Rathish Das, John Iacono and Yakov Nekrich  

### 2022/08/31 [LATIN 2022](https://pakal.cs.cinvestav.mx/latin2022/accepted_latin_2022.html)

1. Elastic Degenerate String Matching with 1 Edit  
Giulia Bernardini, Estéban Gabory, Solon Pissis, Leen Stougie,  Michelle Sweering and Wiktor Zuba  
2. String Attractors and Infinite Words  
Antonio Restivo, Giuseppe Romana and Marinella Sciortino  
3. Near-Optimal Search Time in $\delta$-Optimal Space  
Tomasz Kociumaka, Gonzalo Navarro and Francisco Olivares  
4. Space-efficient data structure for next/previous larger/smaller value queries  
Seungbum Jo and Geunho Kim  
5. Patterns in ordered (random) matchings  
Andrzej Dudek, Jaroslaw Grytczuk and Andrzej Rucinski  

### 2022/07/04 [FOCS 2022](https://focs2022.eecs.berkeley.edu/accepted_papers.html)

1. Online List Labeling: Breaking the $\log^2 n$ Barrier.  
Michael A. Bender (Stony Brook University), Alex Conway (VMWare Research), Martin Farach-Colton (Rutgers University), Hanna Komlos (Rutgers University), William Kuszmaul (MIT), Nicole Wein (DIMACS).  
2. Estimating the Longest Increasing Subsequence in Nearly Optimal Time.  
Alexandr Andoni (Columbia University), Negev Shekel Nosatzki (Columbia University), Sandip Sinha (Columbia University), Clifford Stein (Columbia University).  
3. Õ(n + poly(k))-time Algorithm for Distance k Tree Edit Distance.  
Debarati Das (Penn State University), Jacob Gilbert (University of Maryland), MohammadTaghi Hajiaghayi (University of Maryland), Tomasz Kociumaka (UC Berkeley), Barna Saha (University of California San Diego), Hamed Saleh (University of Maryland).  
4. Faster Pattern Matching under Edit Distance.  
Panagiotis Charalampopoulos (Reichman University, Herzliya, Israel and Birkbeck, University of London), Tomasz Kociumaka (UC Berkeley and Max Planck Institute for Informatics, SIC, Germany), Philip Wellnitz (Max Planck Institute for Informatics, SIC, Germany).  
5. Gap Edit Distance via Non-Adaptive Queries: Simple and Optimal.  
Elazar Goldenberg (The Academic College of Tel Aviv-Yafo), Tomasz Kociumaka (UC Berkeley), Robert Krauthgamer (Weizmann Institute of Science), Barna Saha (University of California San Diego).  
6. Strong XOR Lemma for Communication with Bounded Rounds.  
Huacheng Yu (Princeton University).  

### 2022/06/21 [MFCS 2022](https://www.ac.tuwien.ac.at/mfcs2022/accepted/)

1. Streaming word problems  
Markus Lohrey and Lukas Lück  
2. On Uniformization in the Full Binary Tree  
Alexander Rabinovich  

### 2022/06/18 [ESA 2022](https://algo2022.eu/esa/#papers)

1. An Improved Algorithm for Finding the Shortest Synchronizing Words  
Marek Szykuła and Adam Zyzik.  
2. Lyndon Arrays Simplified  
Jonas Ellert.  
3. Simple worst-case optimal adaptive prefix-free coding  
Travis Gagie.  
4. Approximate Circular Pattern Matching  
Panagiotis Charalampopoulos, Tomasz Kociumaka, Jakub Radoszewski, Solon Pissis, Wojciech Rytter, Tomasz Walen and Wiktor Zuba.  
5. Distinct Elements in Streams: An Algorithm for the (Text) Book  
Sourav Chakraborty, Kuldeep S. Meel and N.V. Vinodchandran.   
6. Insertion Time of Random Walk Cuckoo Hashing below the Peeling Threshold (extended abstract)  
Stefan Walzer.   
7. Approximating Dynamic Time Warping Distance Between Run-Length Encoded Strings  
Zoe Xi and William Kuszmaul.   
8. Computing NP-hard Repetitiveness Measures via MAX-SAT  
Hideo Bannai, Keisuke Goto, Masakazu Ishihata, Shunsuke Kanda, Dominik Köppl and Takaaki Nishimoto.   

### 2022/06/14 [WABI 2022](https://algo2022.eu/wabi/)
1. Haplotype Threading Using the Positional Burrows-Wheeler Transform   
Ahsan Sanaullah, Degui Zhi and Shaojie Zhang.   
2. On Weighted K-Mer Dictionaries  
Giulio Ermanno Pibiri.   
3. Fast gapped k-mer counting with subdivided multi-way bucketed Cuckoo hash tables  
Jens Zentgraf and Sven Rahmann.  
4. Prefix-free parsing for building large tunnelled Wheeler graphs  
Adrián Goga and Andrej Baláž.  
5. Suffix sorting via matching statistics  
Zsuzsanna Liptak, Francesco Masillo and Simon J. Puglisi.  
6. phyBWT: Alignment-free phylogeny via eBWT positional clustering  
Veronica Guerrini, Alessio Conte, Roberto Grossi, Gianni Liti, Giovanna Rosone and Lorenzo Tattini.  
7. Eulertigs: minimum plain text representation of k-mer sets without repetitions in linear time  
Sebastian Schmidt and Jarno Alanko.  
8. Toward optimal fingerprint indexing for large scale genomics  
Clément Agret, Bastien Cazaux and Antoine Limasset.  

## Links
[arXivTimes](https://github.com/arXivTimes/arXivTimes)