# Data Structures and Algorithms: 2016/4/08-14  
1: [On Fully Dynamic Graph Sparsifiers](https://doi.org/10.48550/arXiv.1604.02094)  
2: [Simultaneous Nearest Neighbor Search](https://doi.org/10.48550/arXiv.1604.02188)  
3: [A Primal-Dual Type Algorithm with the $O(1/t)$ Convergence Rate for  Large Scale Constrained Convex Programs](https://doi.org/10.48550/arXiv.1604.02216)  
4: [Two-level lot-sizing with inventory bounds](https://doi.org/10.48550/arXiv.1604.02278)  
5: [The Salesman's Improved Paths: 3/2+1/34 Integrality Gap and  Approximation Ratio](https://doi.org/10.48550/arXiv.1604.02486)  
6: [Computing Longest Increasing Subsequence Over Sequential Data Streams](https://doi.org/10.48550/arXiv.1604.02552)  
7: [An algorithm for 3-SAT problems](https://doi.org/10.48550/arXiv.1604.03462)  
8: [Multi-State Perfect Phylogeny Mixture Deconvolution and Applications to  Cancer Sequencing](https://doi.org/10.48550/arXiv.1604.02605)  
9: [On an algorithm for receiving Sudoku matrices](https://doi.org/10.48550/arXiv.1604.02691)  
10: [An Experimental Study of Dynamic Dominators](https://doi.org/10.48550/arXiv.1604.02711)  
11: [Efficiently Enumerating Minimal Triangulations](https://doi.org/10.48550/arXiv.1604.02833)  
12: [Solving the Team Orienteering Problem with Cutting Planes](https://doi.org/10.48550/arXiv.1604.02934)  
13: [Efficient FPT algorithms for (strict) compatibility of unrooted  phylogenetic trees](https://doi.org/10.48550/arXiv.1604.03008)  
14: [Know When to Persist: Deriving Value from a Stream Buffer](https://doi.org/10.48550/arXiv.1604.03009)  
15: [Amortized Dynamic Cell-Probe Lower Bounds from Four-Party Communication](https://doi.org/10.48550/arXiv.1604.03030)  
16: [Fast Computation of Isomorphisms Between Finite Fields Using Elliptic  Curves](https://doi.org/10.48550/arXiv.1604.03072)  
17: [Efficient Index Maintenance Under Dynamic Genome Modification](https://doi.org/10.48550/arXiv.1604.03132)  
18: [Efficient Parallel Algorithms for k-Center Clustering](https://doi.org/10.48550/arXiv.1604.03228)  
19: [Separating Sets of Strings by Finding Matching Patterns is Almost Always  Hard](https://doi.org/10.48550/arXiv.1604.03243)  
20: [Complexes Detection in Biological Networks via Diversified Dense  Subgraphs Mining](https://doi.org/10.48550/arXiv.1604.03244)  
21: [Accurate, Fast and Lightweight Clustering of de novo Transcriptomes  using Fragment Equivalence Classes](https://doi.org/10.48550/arXiv.1604.03250)  
22: [The Right Mutation Strength for Multi-Valued Decision Variables](https://doi.org/10.48550/arXiv.1604.03277)  
23: [Typical Stability](https://doi.org/10.48550/arXiv.1604.03336)  
24: [Joint alignment of multiple protein-protein interaction networks via  convex optimization](https://doi.org/10.48550/arXiv.1604.03482)  
25: [Ramanujan Graphs in Polynomial Time](https://doi.org/10.48550/arXiv.1604.03544)  
26: [FSG: Fast String Graph Construction for De Novo Assembly of Reads Data](https://doi.org/10.48550/arXiv.1604.03587)  
27: [Dynamic Planar Voronoi Diagrams for General Distance Functions and their  Algorithmic Applications](https://doi.org/10.48550/arXiv.1604.03654)  
28: [A Discrete and Bounded Envy-Free Cake Cutting Protocol for Any Number of  Agents](https://doi.org/10.48550/arXiv.1604.03655)  
29: [Sublinear Time Estimation of Degree Distribution Moments: The Degeneracy  Connection](https://doi.org/10.48550/arXiv.1604.03661)  
30: [Optimal Monotone Drawings of Trees](https://doi.org/10.48550/arXiv.1604.03921)  
31: [Lossy Kernelization](https://doi.org/10.48550/arXiv.1604.04111)  